code/modules/mob/living/simple_animal/abnormality/aleph/white_night.dm

```diff
@@ -313,11 +322,14 @@ GLOBAL_LIST_EMPTY(apostles)
 	maxHealth = 3000
 	speed = 5
 	move_to_delay = 7
+	melee_damage_lower = 70 // However, make them do more damage in melee to compensate.
```

Comment:
That's too much for a pale damage, and all apostles are meant to damage with their abilities, not melee.

Reply:
I change it so they have slightly more frequent AoEs